roles/ocluster/templates/worker

```diff
@@ -7,13 +7,22 @@
 . /etc/rc.subr
 
 name=worker
+title=ocluster_worker # Title of the process
+# The command to invoke the worker process
+worker_command="/usr/local/bin/${name} --connect /etc/ocluster/pool.cap --name {{ inventory_hostname_short }} --state-dir /obuilder/state --obuilder-store zfs:obuilder --obuilder-healthcheck 0 --obuilder-prune-threshold=25 --capacity={{ capacity }} -v"
+
 rcvar=worker_enable
 
 PATH=$PATH:/usr/local/bin
 export PATH
 
-command="/usr/local/bin/${name}"
-command_args="--connect /etc/ocluster/pool.cap --name {{ inventory_hostname_short }} --state-dir /obuilder/state --obuilder-store zfs:obuilder --obuilder-healthcheck 0 --obuilder-prune-threshold=25 --capacity={{ capacity }} -v 2>&1 | logger &"
+# `daemon` will run the worker process
+command="/usr/sbin/daemon"
+# --restart : supervise the process and restart it after 1 second if it crashes
+# --syslog : direct stdout and stderr to the syslog
+# --title : the name to give the process
+# See https://man.freebsd.org/cgi/man.cgi?daemon(8)
+command_args="--restart --syslog --title ${title} ${worker_command}"
```

Comment:
@mtelvers has suggested we should have a slower restart time: 60secs

Reply:
https://github.com/ocurrent/freebsd-infra/pull/5/commits/d0890111eaa93a936ce086745a975e3c573b8c11